ping_app/periods_service.py

```diff
@@ -46,18 +52,34 @@ def get_periods_by_date(self, start_date: datetime, finish_date: datetime):
         return periods
 
     @staticmethod
-    def convert_date_string(datesting: str) -> datetime:
-        return datetime.strptime(datesting, "%Y-%m-%d")
+    def count_period_duration(period: Period, search_date: datetime) -> timedelta:
```

Comment:
According to name it shouldn't be hard to get period from `Period`, but looks like naming is not good enough here 😅